chain/client/src/client_actor.rs

```diff
@@ -438,7 +439,18 @@ impl Handler<NetworkClientMessages> for ClientActor {
             NetworkClientMessages::AnnounceAccount(announce_accounts) => {
                 let mut filtered_announce_accounts = Vec::new();
 
-                for announce_account in announce_accounts.into_iter() {
+                for (announce_account, last_epoch) in announce_accounts.into_iter() {
```

Comment:
`into_iter` is unnecessary

Reply:
I need the object since I'm putting it back to `filtered_announce_accounts`. This iteration is like a `filter` but it has a `return` in one of the options. That was the reason I iterate instead of `filter`.

Reply:
Yeah but how is that related to using `into_iter`? It is semantically the same as not using it here.